### ISSUE TITLE
Include missing information in the Activities Instruction view

### ISSUE THREAD

**Issue by rsg73626** [ecureuill/SELIProject#82]
When an activity is rendered, in the student side, it shows some important informations. But when the student clicks to open the activity (a modal), this important information is not presented in the modal. It can created difficulties for students with cognitive disabilities.

## Expected Behavior

All the informations must be presented in both of the screens. 

## Current Behavior

The informations are only presented in the first screen.

## Steps to Reproduce

Click in some student activity in the student area.

**[SELI - Accessibility test](https://docs.google.com/presentation/d/1OBRtT9S3NZWK2bq9wvEzXLxcoQMxMC_CesAsUgqkUww/edit#slide=id.g656ec9906d_4_91) (slides 30-31)**

## Environment

Browser: all 

## Possible Solution

Add the informations to both the screens as indicated in the slides.

**Comment by ecureuill** [comment]
https://github.com/solex2006/SELIProject/issues/36
